firebase-vertexai/src/main/kotlin/com/google/firebase/vertexai/type/Voices.kt

```diff
@@ -27,10 +27,15 @@ public class Voices private constructor(public val ordinal: Int) {
 
```

Comment:
We default to `Voices.PUCK`. We should support `UNSPECIFIED` as well, and throw an error like our other enums when it's not supported.

Reply:
the backend model defaults to puck if nothing is specified and hence the default as puck here